bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/constants/RuntimeConstants.java

```diff
@@ -77,6 +72,14 @@
     public static final String FLOAT_LANG_LIB = "lang.float";
     public static final String BOOLEAN_LANG_LIB = "lang.boolean";
     public static final String TRANSACTION_LANG_LIB = "lang.transaction";
+    public static final String RUNTIME_LANG_LIB = "lang.runtime";
+
+    public static final Module BALLERINA_BUILTIN_PKG_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, "builtin");
```

Comment:
is there a reason this is moved down?

Reply:
Not for any technical reasons. just wanted to keep the module names above since the module name strings are used at module definitions (it may bring better look)